p/themes/Swage/swage.scss

```diff
@@ -746,6 +746,14 @@ form {
 	}
 }
 
+.reader #new-article {
+	position: fixed;
+    top: 35px;
+    width: 100%;
+    padding: 10px 0;
+	bottom: unset;
+}
+
```

Comment:
Mix of tabs and spaces ;-)

Reply:
Was that the only theme requiring some adjustments?

Reply:
It is, because the other themes don't muck with the positioning of that div the way I do with Swage.

Reply:
Also, re:mixing tabs and spaces: woops. This is what happens when I switch editors mid-stream.

Reply:
By the way, we have a https://github.com/FreshRSS/FreshRSS/blob/dev/.editorconfig declaration
See https://editorconfig.org